testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkCompatibility.java

```diff
@@ -590,7 +590,7 @@ public void testSparkReadingCompositeTrinoData(StorageFormat storageFormat)
         onTrino().executeQuery("DROP TABLE " + trinoTableName);
     }
 
-    @Test(groups = {ICEBERG, PROFILE_SPECIFIC_TESTS, ICEBERG_REST, ICEBERG_JDBC, ICEBERG_NESSIE}, dataProvider = "storageFormatsWithSpecVersion")
```

Comment:
Nessie will have extra table properties from `NessieUtil#updateTableMetadataWithNessieSpecificProperties()` https://github.com/apache/iceberg/blob/385a6bd296e551bdfdab48d2a291d3516f999f5a/nessie/src/main/java/org/apache/iceberg/nessie/NessieUtil.java#L137

So, this generic table property check will fail. Hence, removed the Nessie group from this as I didn't see any group specific check in the testcase.